client/account-recovery/forgot-username/forgot-username-form/test/index.jsx

```diff
@@ -34,18 +32,11 @@ describe( 'ForgotUsername', () => {
 	context( 'fields', () => {
 		useFakeDom();
 
-		it( 'should be disabled while submitting', function() {
-			const wrapper = mount( <ForgotUsernameFormComponent className="test__test" /> );
-
-			// Initialize all inputs with some dummy text
-			inputSelectors.forEach( selector => {
-				wrapper.find( selector ).node.value = 'dummy text';
-				wrapper.find( selector ).simulate( 'change' );
-			} );
-
-			wrapper.find( '.forgot-username-form__submit-button' ).simulate( 'click' );
-
-			expect( wrapper ).to.have.state( 'isSubmitting' ).to.be.true;
+		it( 'should be disabled when isRequesting is on', function() {
+			const wrapper = mount(
+				<ForgotUsernameFormComponent className="test__test"
+					isRequesting={ true }
+				/> );
```

Comment:
Not at all. Given the fact that the tested component itself is arranging properties in this way, so should the test code. Fixed by c54e2903eed0c14b4d3667f90ba423528962aa09.